db/migrate/20240816163531_add_fuzzystrmatch_extension.rb

```diff
@@ -0,0 +1,5 @@
+class AddFuzzystrmatchExtension < ActiveRecord::Migration[7.1]
```

Comment:
Just a quick note: this will have to be moved into its own branch. Migrations have to be merged in separately.

Reply:
That's good to know!